### PR TITLE
Releases Page Update

### PR DESCRIPTION
**Pull request by jongio**
You can view a preview of the new page here: https://aka.ms/azsdk/preview

- Include new page header text
- Change layout to 4 columns. Removed Notes column
- Include "Next Stable" column
- Include "(New)" in tabs and display name on "All" tab
- Change Code/Docs badges to links
- Include link to Support
- Include "Replaced By" in name column in All tab.
- Include package name under display name in Name column

@weshaggard - I will remove the Action deployment file before we merge. Keeping of now just in case there are more edits we want to preview before merging.

### CODE REVIEW

File: _includes/releases/links.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% assign version = item[include.version] %}
+
+{% if version != "" %}
+
```
Comment (weshaggard) [inline _includes/releases/links.md:4]: nit: multiple empty lines.

File: _includes/releases/links.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        {% assign msdocs_url = item.MSDocs %}
+        {% if item.MSDocs == "" %}
+            {% assign msdocs_url = msdocs_url_template | replace: 'item.Package', item.Package | replace: 'item.TrimmedPackage', trimmedPackage %}
+            {% if item.VersionGA == "" and item.VersionPreview != "" and pre_suffix.size > 0 %}
+                {% assign msdocs_url = msdocs_url | append: pre_suffix %}
+            {% endif %}
+        {% endif %}
+        {% assign msdocs_url = msdocs_url | replace: 'item.Version', version %}
+        {% if code_url != "" and code_url != "NA" and msdocs_url != "" and msdocs_url != "NA" %}
+            &nbsp;|&nbsp;
```
Comment (weshaggard) [inline _includes/releases/links.md:32]: Is it worth having these side-by-side? Maybe just always have them in a vertical list. Just a minor suggestion I don't mind either way.

**Review by weshaggard — APPROVED** [review]

File: _includes/releases/links.md (hunk 2 above)
Comment (jongio) [reply _includes/releases/links.md:32]: I would like to save vertical space on this one.

**Comment by weshaggard** [comment]
```
'file:///home/vsts/work/1/s/azure-sdk/_site/releases/latest/all/dotnet.html' has 4 broken link(s):
  https://www.nuget.org/packages/ Azure.Security.KeyVault.Certificates
  https://www.nuget.org/packages/ Azure.Security.KeyVault.Secrets
  https://www.nuget.org/packages/ Azure.Storage.Files.Shares
  https://www.nuget.org/packages/ Azure.Storage.Queues
```
Looks like there might be some extra spaces coming in some where.

**Comment by weshaggard** [comment]
Looks like there is a space after the comma in the csv file. So we either need to trim the space or remove it in the csv.

**Review by elraikhm — APPROVED** [review]
Looks good

**Pull request merged**